rhemyn/compile.js

```diff
@@ -16,11 +16,30 @@ const generate = (node, negated = false, get = true, stringSize = 2, func = 'tes
   let out = [];
   switch (node.type) {
     case 'Expression':
+      let succceedsZero = true;
+      let queue = [...node.body];
+      let n = queue.shift()
```

Comment:
Can we do a recursive loop/func here instead of a queue? Although probably fine either way.